sound/soc/sof/imx/imx8.c

```diff
@@ -607,7 +607,22 @@ static struct snd_sof_dsp_ops sof_imx8x_ops = {
 			SNDRV_PCM_INFO_NO_PERIOD_WAKEUP
 };
 
```

Comment:
In commit message, there's a typo: s/compatbile/compatible. Also, checkpatch noticed it.

Reply:
Thanks! Fixed.